project/RootSettings.scala

```diff
@@ -5,18 +5,10 @@ package akka
 
 import sbt._
 import sbt.Keys._
-import akka.ValidatePullRequest.validatePullRequest
-import sbtunidoc.Plugin.UnidocKeys.unidoc
-import com.typesafe.sbt.site.SphinxSupport
-import com.typesafe.sbt.site.SphinxSupport.Sphinx
 
 object RootSettings extends AutoPlugin {
 
-  lazy val docs = ProjectRef(file("."), "akka-docs")
-
   // settings for root project goes here
   override val projectSettings = Seq(
-    validatePullRequest <<= validatePullRequest.dependsOn(unidoc in Compile),
-    validatePullRequest <<= validatePullRequest.dependsOn(SphinxSupport.generate in Sphinx in docs)
   )
 }
```

Comment:
Hmm, I can’t seem to make sense of this, please help: my naïve understanding is that removing this plugin means that the root project (`/build.sbt`) will need to explicitly enable the UnidocRoot and TimeStampede plugins. If that is true then I’d prefer that, it would make things more explicit without losing functionality—but I think I’m still missing something.